Nodify/Connections/PendingConnection.cs

```diff
@@ -305,6 +305,9 @@ protected virtual void OnPendingConnectionDrag(object sender, PendingConnectionE
                             {
                                 PreviewTarget = connector.DataContext;
                             }
+                        }else if (EnablePreview)
+                        {
+                            PreviewTarget = null;
```

Comment:
I would remove the nested if statement and assign `PreviewTarget = connector?.DataContext` for readability.

```csharp
// And we have a connector
if (connector != null)
{
    SetIsOverElement(connector, true);
}

// Update the preview target if enabled
if (EnablePreview)
{
    PreviewTarget = connector?.DataContext;
}
```